ext/pgsql/tests/pg_fetch_object_with_abstract_class.phpt

```diff
@@ -50,8 +50,8 @@ try {
 --CLEAN--
 <?php
 include('config.inc');
-$db = pg_connect($conn_str);
-pg_query($db, "DROP TABLE IF EXISTS pg_fetch_object_abstract_class cascade");
+$db = @pg_connect($conn_str);
```

Comment:
This issue really weirds me out. would it be possible to do the cleaning somewhere in the file section instead just for testing once at least ?